pkgs/top-level/all-packages.nix

```diff
@@ -40373,6 +40373,8 @@ with pkgs;
 
   djenrandom = callPackage ../tools/misc/djenrandom { };
 
+  djent = callPackage ../by-name/dj/djent/package.nix { };
```

Comment:
Now that the package is in `by-name` we don't need this anymore. The package attribute is automatically constructed. One of the compelling benefits of the by-name approach is that we don't need all-packages anymore.

Reply:
I have now removed the line. Thanks for the advice.